### PR TITLE
metrics(replacer): Consumer group per replacement processed

### PR DESCRIPTION
**Pull request by rahul-kumar-saini**
- We're adding a new consumer group for replacements
- We'd like to see replacements metrics split by consumer groups and update some alerts

### CODE REVIEW

File: snuba/datasets/errors_replacer.py [hunk 1]
```diff
@@ -485,7 +485,11 @@ def process_message(self, message: ReplacementMessage) -> Optional[Replacement]:
         type_ = message.action_type
 
         if type_ in REPLACEMENT_EVENT_TYPES:
-            metrics.increment("process", 1, tags={"type": type_})
+            metrics.increment(
```
Comment (nikhars) [inline snuba/datasets/errors_replacer.py:488]: Is this the only metric which is relevant per consumer group?

**Review by nikhars — CHANGES_REQUESTED** [review]
Can you add the consumer group as a tag over here https://github.com/getsentry/snuba/blob/master/snuba/cli/replacer.py#L102. That way any metrics reported by the ReplacerWorker will have the relevant tags

**Comment by rahul-kumar-saini** [comment]
> Can you add the consumer group as a tag over here https://github.com/getsentry/snuba/blob/master/snuba/cli/replacer.py#L102.

I think this already has the consumer group:
https://github.com/getsentry/snuba/blob/master/snuba/cli/replacer.py#L93

File: snuba/datasets/errors_replacer.py (hunk 1 above)
Comment (rahul-kumar-saini) [reply snuba/datasets/errors_replacer.py:488]: I believe so. The other metrics already have consumer group data.

**Review by nikhars — APPROVED** [review]

**Pull request merged**